apps/client/src/components/Conclusion/NewCheckerModal.tsx

```diff
@@ -87,6 +87,7 @@ const NewCheckerModal: React.FC<{
             <Radio
               checked={saveAddress === true}
```

Comment:
I thought we said we're going to enable yes on default?

Reply:
No we leave it empty, but force the user to chose buy giving an error when leaving empty